firmware/Core/Src/main.c

```diff
@@ -82,7 +83,7 @@ const osThreadAttr_t TASK_DEBUG_print_heartbeat_Attributes = {
 osThreadId_t TASK_handle_uart_telecommands_Handle;
 const osThreadAttr_t TASK_handle_uart_telecommands_Attributes = {
   .name = "TASK_handle_uart_telecommands",
-  // Size 2048 doesn't work with LFS settings, but 8192 does
+  // Size 2048 doesn't work with LFS settings, but 8192 does.
   // TODO: confirm stack size
   .stack_size = 8192,
```

Comment:
Suggest to `#define` the stack size.

Reply:
Stack size is only referenced in one location. Unnecessary abstraction in my opinion. Each FreeRTOS task has its own stack size, so defining it right with the task seems reasonable.